dolt/mysql.js

```diff
@@ -260,7 +261,9 @@ export async function addChallenges(connection, data) {
           continue;
         }
       } else if (columnType === 'JSON') {
-        values.push(JSON.stringify(value));
+        // to clone the object so we can delete the null values
+        const newValue = JSON.parse(JSON.stringify(value));
+        values.push(JSON.stringify(removeNulls(newValue)));
```

Comment:
I am fine with leaving this. However, this was the "quick and dirty" implementation for the PoC.

I assumed, at some point, we would split the `JSON` values up into their own tables.

Reply:
lets go with it for now then - it's fairly inconsequential I think, we can always remove it later.